protocol/protocol.tex

```diff
@@ -9078,146 +11147,319 @@
+\vspace{-2.5ex}
+\begin{center}
+\scalebox{0.7}{
+\notsprout{\renewcommand{\arraystretch}{1.28}}
+\hbadness=10000
+\begin{tabularx}{1.41\textwidth}{|c|c|l|p{11.1em}|L|}
+\hline
+\!\!Note\!\! & \heading{Bytes} & \heading{Name} & \heading{Data Type} & \heading{Description} \\
+\hhline{|=|=|=|=|=|}
+
+& $4$ & $\headerField$ & \type{uint32} & Contains: \begin{compactitemize}
+                                                    \item $\fOverwintered$ flag (bit $31$, always set)
+                                                    \item $\versionField$ (bits $\barerange{30}{0}$) --
+                                                          \transactionVersion.
+                                                  \end{compactitemize} \\ \hline
+
+& $4$ & $\nVersionGroupId\!$ & \type{uint32} & Version group ID (nonzero). \\ \hline
+
+& $4$ & $\lockTime$ & \type{uint32} & Unix-epoch UTC time or \blockHeight, encoded as in \Bitcoin. \\ \hline
+
+& $4$ & $\nExpiryHeight$ & \type{uint32} &
+A \blockHeight in the range $\range{1}{499999999}$ after which the \transaction will expire, or $0$ to disable expiry.
+\smash{\cite{ZIP-203}} \\
+\hhline{|=|=|=|=|=|}
+
+& \Varies & $\txInCount$ & \type{compactSize} & Number of \transparent inputs. \\ \hline
+
+& \Varies & $\txIn$ & $\txIn$ & \xTransparent inputs, encoded as in \Bitcoin. \\ \hline
+
+& \Varies & $\txOutCount$ & \type{compactSize} & Number of \transparent outputs. \\ \hline
+
+& \Varies & $\txOut$ & $\txOut$ & \xTransparent outputs, encoded as in \Bitcoin. \\
+\hhline{|=|=|=|=|=|}
+
+$\footnotestar$ & \Varies & $\nJoinSplit$ & \type{compactSize} &
+The number of \joinSplitDescriptions in $\vJoinSplit$. \\ \hline
+
+$\footnotestar$ & \Longunderstack{$1698 \mult$ \\ $\nJoinSplit$} & $\vJoinSplit$ & \type{JSDescriptionGroth16}\!\! \type{[$\nJoinSplit$]} &
+A sequence of \joinSplitDescriptions using \Groth proofs, encoded per \crossref{joinsplitencodingandconsensus}. \\ \hline
+
+$\footnotestar\;\dagger$ & $32$ & $\joinSplitPubKey\!$ & \type{byte[32]} &
+An encoding of a $\JoinSplitSig$ public \validatingKey. \\ \hline
+
+$\footnotestar\;\dagger$ & $64$ & $\joinSplitSig$ & \type{byte[64]} &
+A signature on a prefix of the \transaction encoding, validated using $\joinSplitPubKey$ as specified in
+\crossref{sproutnonmalleability}. \\
+\hhline{|=|=|=|=|=|}
+
+& \Varies & $\nSpendsSapling$ & \type{compactSize} &
+The number of \spendDescriptions in $\vSpendsSapling$. \\ \hline
+
+& \Longunderstack{$362 \mult$ \\$\!\nSpendsSapling\!$} & $\vSpendsSapling$ & \type{SpendDescriptionV5} \type{[$\nSpendsSapling$]} &
+A sequence of \spendDescriptions{}, encoded per \crossref{spendencodingandconsensus}. \\ \hline
+
+& \Varies & $\nOutputsSapling\!$ & \type{compactSize} &
+The number of \outputDescriptions in $\vOutputsSapling$. \\ \hline
+
+& \Longunderstack{$948 \mult$ \\$\!\nOutputsSapling\!$} & $\vOutputsSapling\!$ & \type{OutputDescriptionV5} \type{[$\nOutputsSapling$]} &
```

Comment:
Since most fields are moved elsewhere, `SpendDescriptionV5` is somewhat smaller:
```suggestion
& \Longunderstack{$756 \mult$ \\$\!\nOutputsSapling\!$} & $\vOutputsSapling\!$ & \type{OutputDescriptionV5} \type{[$\nOutputsSapling$]} &
```